src/richie/apps/courses/templates/courses/cms/fragment_course_glimpse.html

```diff
@@ -37,12 +37,12 @@
                     {% endget_page_plugins %}
                     {% blockplugin plugins.0 %}
                         <div class="course-glimpse__organization-logo">
-                            <img src="{% thumbnail instance.picture 200x113 upscale subject_location=instance.picture.subject_location %}"
+                            <img src="{% thumbnail instance.picture 200x113 replace_alpha='#FFFFFF' upscale subject_location=instance.picture.subject_location %}"
                                 srcset="
                                     {% thumbnail instance.picture 200x113 upscale subject_location=instance.picture.subject_location %} 200w
```

Comment:
Missed one
```suggestion
                                    {% thumbnail instance.picture 200x113 replace_alpha='#FFFFFF' upscale subject_location=instance.picture.subject_location %} 200w
```

Reply:
Good catch :muscle:!
I corrected it so I hope now everything works :pray: